src/planscape/conditions/views.py

```diff
@@ -20,7 +21,7 @@
 # Global variable for the ColormapConfig, so that the configuration file is read once.
 PLANSCAPE_ROOT_DIRECTORY = cast(str, cfg('PLANSCAPE_ROOT_DIRECTORY'))
```

Comment:
I think we can delete this line?  And lines 4 and 7 then too?

Reply:
makes sense! I've removed those lines and the demo still works.